internal/report/report.go

```diff
@@ -73,7 +73,7 @@ func AddResultToPolicyReport(
 }
 
 // NewClusterPolicyReport creates a new ClusterPolicyReport from a given resource
-func NewClusterPolicyReport(runUID string, resource unstructured.Unstructured) *wgpolicy.ClusterPolicyReport {
+func NewClusterPolicyReport(runUID string, resource unstructured.Unstructured, skipppedPoliciesNum, erroredPoliciesNum int) *wgpolicy.ClusterPolicyReport {
```

Comment:
we could leave the constructor as is and use the fields directly

Reply:
Indeed, dunno what I was thinking. Changed, which removes quite the unneeded changes.